hack/tools/Makefile

```diff
@@ -99,7 +99,7 @@ golangci-lint: $(GOLANGCI_LINT)
 $(GOLANGCI_LINT): $(REPO_ROOT)/.github/workflows/golangci-lint.yaml
 	$(REPO_ROOT)/hack/ensure-golangci-lint.sh \
 		-b $(BIN_DIR) \
-		$(shell cat .github/workflows/golangci-lint.yaml | grep version | sed 's/.*version: //')
```

Comment:
Let's drop this from this PR. This was changed on main to a totally different implementation
(also not sure if we can assume that yq is always available)